DigitalLearningSolutions.Web/ViewModels/Common/ViewComponents/SetPasswordViewModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.Common.ViewComponents
+{
+    public class SetPasswordViewModel : ConfirmPasswordViewModel
+    {
+        public SetPasswordViewModel(
+            string label = "Password"

```

Comment:
Feels a bit weird that you're constructing a separate confirm password view model rather than taking in the one that the view model has.
Because we don't send passwords back to the user for security's sake, it doesn't make a difference, but if we did, I think this setup would stop dotnet from populating the input fields with the data we've returned.

Reply:
(I think this should be changed)

Reply:
Done